migration/iD11/oracle/202309291327_IDEMPIERE-5836.sql

```diff
@@ -0,0 +1,16 @@
+SELECT register_migration_script('202309291327_IDEMPIERE-5836.sql') FROM dual;
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
```

Comment:
The comment on line 5 is wrong "I forgot to set the DICTIONARY_ID_COMMENTS System Configurator" better to delete it or make it correct